test/e2e/parallel/spacerequest_test.go

```diff
@@ -137,7 +144,16 @@ func TestCreateSpaceRequest(t *testing.T) {
 			// then
 			// subSpace should be deleted as well
 			require.NoError(t, err)
-			err = memberAwait.WaitUntilNamespaceDeleted(t, subSpace.Name, "appstudio")
+			// check that created namespaces secret access are deleted
+			for _, nsAccess := range spaceRequest.Status.NamespaceAccess {
+				err = memberAwait.WaitUntilSecretDeleted(t, &corev1.Secret{
+					ObjectMeta: metav1.ObjectMeta{Name: nsAccess.SecretRef, Namespace: spaceRequest.GetNamespace()},
+				})
+				require.NoError(t, err)
+			}
```

Comment:
another approach would be checking that there is no secret with 
```
toolchain.dev.openshift.com/spacerequest
```
label

Reply:
I agree, It's probably better with the labels and `client.List` call. I've done some refactor in https://github.com/codeready-toolchain/toolchain-e2e/pull/706/commits/71e2a1a5e56b0a13eb6507f0f1cb186ab6816617

But I'm still unsure if `testsupport/wait/member.go` is a good place for `WaitUntilSecretsDeleted`, since it's not listing in the member operator namespace. Any suggestion on where to move this function? 
Maybe we can move it later.

Reply:
Yeah, it could have been moved up to the `Awaitility` type since it accepts the target namespace anyway :+1:
Feel free to do it either here or in a separate PR :wink: